navigation/navigation-common/build.gradle

```diff
@@ -55,12 +55,12 @@ androidXMultiplatform {
             dependencies {
                 api "org.jetbrains.compose.annotation-internal:annotation:1.7.1"
                 api "org.jetbrains.compose.collection-internal:collection:1.7.1"
-                api project(":core:core-bundle")
-                api project(":core:core-uri")
-                api project(":lifecycle:lifecycle-common")
-                api project(":lifecycle:lifecycle-runtime")
-                api project(":lifecycle:lifecycle-viewmodel")
-                api project(":lifecycle:lifecycle-viewmodel-savedstate")
+                api "org.jetbrains.androidx.core:core-bundle:1.1.0-alpha02"
+                api "org.jetbrains.androidx.core:core-uri:1.1.0-alpha02"
```

Comment:
Why do we unpin it here?

Reply:
We shouldn't pin to not stable versions

Reply:
The current algorithm of unpinning is to find all project dependencies after a stable release, and replace them by stable version.

If we pin to not stable version, we miss dependencies

Reply:
this is a temporary short term solution before a proper migration to new modules

Reply:
Please, create a Major task with target Compose 1.8.0 to pin to the new stable versions. 

If we won't release new stable versions of these libs with Compose, then we'll change the target to 1.9